rackcity/models/asset.py

```diff
@@ -7,6 +7,13 @@
 from .rack import Rack
 
 
+def get_asset_number():
```

Comment:
I would maybe rename something like "get_next_available_asset_number", sort of a long name but makes it more obvious whats goin on??

Reply:
coool